lib/swoosh/adapters/postmark.ex

```diff
@@ -8,8 +8,9 @@ defmodule Swoosh.Adapters.Postmark do
 
       # config/config.exs
       config :sample, Sample.Mailer,
-        adapter: Swoosh.Adapters.Postmark,
-        api_key: "my-api-key"
+        adapter:  Swoosh.Adapters.Postmark,
+        api_key:  "my-api-key",
+        template: true # optional
```

Comment:
Because constructing endpoint happens earlier - during initialization, not during email delivery. We shouldn't modify endpoint later on but do that when defining an adapter https://github.com/KamilLelonek/swoosh/blob/79e0b7bef1692a9f7ebc92d34a663f16db2d4e1d/lib/swoosh/adapters/postmark.ex#L58-L59